### DISCUSSION TITLE
Unable to join discord server, but account is verified.

### DISCUSSION THREAD

**Discussion by Vymok19** [chromiecraft/chromiecraft#5418] [General]
I was in the discord before, but after leaving it a while ago and joining back, I'm unable to react to the message that allows me to see the channels.

**Comment by 55Honey** [comment]
Hello Vymok19,
To access all channels please see the #welcome channel.
Click the white check mark reaction, so you gain the verified role.

**Reply by Vymok19** [reply]
Hello,

There is no way for me to react to the message in the channel. No icons outside of the frame, and even right clicking doesn't show a react command.

**Reply by Vymok19** [reply]
![Capture](https://user-images.githubusercontent.com/130678692/232141570-037328e6-9868-455a-ba92-74a6bfce1ae8.PNG)

This is what is showing on the server. I can see reactions to the message, but I don't have an option to react anywhere.

**Reply by 55Honey** [reply]
Did you ignore the MEE6 bot? Or are you using a custom client?
Is your account verified with phone verification?
